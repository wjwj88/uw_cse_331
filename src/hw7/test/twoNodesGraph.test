# Find a path with Dijkstra's algorithm when there are two possible paths hw7

CreateGraph characters
AddNode characters GORILLA-MAN
AddNode characters HAWK
AddEdge characters GORILLA-MAN HAWK 1
AddEdge characters GORILLA-MAN HAWK 0.999

FindPath characters GORILLA-MAN HAWK