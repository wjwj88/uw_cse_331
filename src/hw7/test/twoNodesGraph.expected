# Find a path with Dijkstra's algorithm when there are two possible paths hw7

created graph characters
added node GORILLA-MAN to characters
added node HAWK to characters
added edge 1.000 from GORILLA-MAN to HAWK in characters
added edge 0.999 from GORILLA-MAN to HAWK in characters

path from GORILLA-MAN to HAWK:
GORILLA-MAN to HAWK with weight 0.999
total cost: 0.999